ui/app/AppLayouts/Communities/panels/SortableTokenHoldersList.qml

```diff
@@ -181,14 +190,31 @@ StatusListView {
         background: Item {
             Rectangle {
                 anchors.fill: parent
-
                 anchors.topMargin: delegate.topPadding
 
                 radius: Style.current.radius
                 color: (delegate.hovered || delegate.ListView.isCurrentItem)
                        ? Theme.palette.baseColor2 : "transparent"
             }
 
+            Rectangle {
+                anchors.fill: parent
+                anchors.topMargin: delegate.topPadding
+
+                radius: Style.current.radius
+                color: "transparent"
+
+                SequentialAnimation on color {
+                    id: colorAnimation
+
+                    running: false
```

Comment:
can't you set `running: model.remotelyDestructState === Constants.ContractTransactionStatus.InProgress`?

Reply:
No, it doesn't work bc start needs to be triggered by change, not presence of given condition. In this case the animation would start immediately when burn is in progress, not when transits from InProgress to different state.